packages/doenetml-worker-rust/lib-doenetml-core/src/core/state/prop.rs

```diff
@@ -335,19 +335,30 @@ impl<T: Default + Clone> Prop<T> {
 
     /// Convenience function to call `calculate` on interface
```

Comment:
-`on interface`

Reply:
old terminology. It's now called the `updater`.